src/store.js

```diff
@@ -7,14 +7,23 @@ import thunk from 'redux-thunk';
 // import the root reducer
 import rootReducer from './js/reducers/index';
 
-// import data
-
+// temporary import data
+const posts = [
+  {
+    "name": "John Doe",
+    "comment": "I wish I was a real person.",
+    "id":"1"
+  },
+  {
+    "name": "James Dean",
+    "comment": "Singing is amazing.",
+    "id":"2"
+  }
+]
```

Comment:
Is this the same format as the response from the API will be?  We should make sure there is parity, as it will make integration MUCH easier.